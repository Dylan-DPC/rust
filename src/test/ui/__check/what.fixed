// check-pass
// run-rustfix
#![warn(unused_parens)]
fn test(_: u32) {}

fn main() {
    test(7);
    //~^^^ WARN unnecessary parentheses around function argument
}
