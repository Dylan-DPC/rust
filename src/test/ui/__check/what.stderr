warning: unnecessary parentheses around function argument
  --> $DIR/what.rs:7:10
   |
LL |       test((
   |  __________^
LL | |         7
LL | |     ));
   | |_____^ help: remove these parentheses
   |
note: the lint level is defined here
  --> $DIR/what.rs:3:9
   |
LL | #![warn(unused_parens)]
   |         ^^^^^^^^^^^^^

warning: 1 warning emitted

